pkgs/stdenv/generic/setup.sh

```diff
@@ -1022,6 +1022,15 @@ configurePhase() {
             fixLibtool "$i"
         done
 
+        # Make sure `find` is the full `find` from findutils, rather
+        # than from busybox or some other implementation.  This is
+        # analogous to `shellcheck shell=bash` at the top of this
```

Comment:
shellcheck is a tool to verify if shell scripts are valid, it's unrelated to checking if the correct shell is used.

Reply:
:embarrassed: